iib/web/models.py

```diff
@@ -886,7 +911,7 @@ def from_json(cls, kwargs, batch=None):
         """
         request_kwargs = deepcopy(kwargs)
 
-        for key in ('bundles', 'deprecation_list'):
+        for key in ('bundles', 'deprecation_list', 'build_tags'):
```

Comment:
Let's check this in Request._from_json

Reply:
I'm not sure build_tags should be available for all requests. Therefore I added it per class

Reply:
Since you have added it under additional_optional_params in `_from_json`, let's have this check in there if the user has provided `build_tags`. That would mean removing it from here.